Compile bug: CUDA build uses too much shared data

### Git commit

b6498

### Operating systems

Linux

### GGML backends

CUDA

### Problem description & steps to reproduce

export CUDA_ARCH="60;61;70;75"

 build

### First Bad Commit

b6498

### Compile command

```shell
cmake
make
```

### Relevant log output

```shell
[ 27%] Building CUDA object ggml/src/ggml-cuda/CMakeFiles/ggml-cuda.dir/pool2d.cu.o
ptxas error   : Entry function '_Z15flash_attn_tileILi128ELi32ELb1EEvPKcS1_S1_S1_S1_PKiPfP6float2ffffjfiiiiiiiiiiiiiliiliiiiil' uses too much shared data (0xc400 bytes, 0xc000 max)
ptxas error   : Entry function '_Z15flash_attn_tileILi128ELi32ELb0EEvPKcS1_S1_S1_S1_PKiPfP6float2ffffjfiiiiiiiiiiiiiliiliiiiil' uses too much shared data (0xc400 bytes, 0xc000 max)
make[3]: *** [ggml/src/ggml-cuda/CMakeFiles/ggml-cuda.dir/build.make:314: ggml/src/ggml-cuda/CMakeFiles/ggml-cuda.dir/fattn-tile.cu.o] Error 255
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/dev/shm/tmp/build'
make[2]: *** [CMakeFiles/Makefile2:1024: ggml/src/ggml-cuda/CMakeFiles/ggml-cuda.dir/all] Error 2
make[2]: Leaving directory '/dev/shm/tmp/build'
make[1]: *** [Makefile:136: all] Error 2
```

--- a/ggml/src/ggml-cuda/fattn-tile.cu
+++ b/ggml/src/ggml-cuda/fattn-tile.cu
@@ -35,7 +35,6 @@ static int fattn_tile_get_kq_stride_host(const int D, const int ncols, const int
         switch (D) {
             case 64:
             case 128:
-                return 128;
             case 256:
                 return ncols <= 16 ? 128 : 64;
             default:
@@ -86,7 +85,6 @@ static constexpr __device__ int fattn_tile_get_kq_stride_device(int D, int ncols
     switch (D) {
         case 64:
         case 128:
-            return 128;
         case 256:
             return ncols <= 16 ? 128 : 64;
         default:

